src/components/List/List.stories.tsx

```diff
@@ -32,19 +32,35 @@ const itemGroups = [
   "Villagers",
 ];
 
-const ListTemplate = ({ listType, items, ...args }) => (
-  <List type={listType} {...args}>
-    {items.map((item, i) => (
-      <li key={i}>{...item}</li>
-    ))}
+const definitions = [
+  {
+    term: "Summary",
+    definition:
+      "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. ",
```

Comment:
Can we get an example of a definition with `<em>` or links?  Or is that impossible in definitions?